magefile.go

```diff
@@ -1196,12 +1196,16 @@ func packageAgent(ctx context.Context, platforms []string, dependenciesVersion s
 		log.Printf("dependencies extracted from package specs: %v", dependencies)
 	}
 
+	keepArchive := os.Getenv("KEEP_ARCHIVE") != ""
```

Comment:
should `KEEP_ARCHIVE` env var also affect the `mage clean` target?

Reply:
I don't think so. `mage clean` is a clear signal that the user wants all the build-related data gone. Although it's true that we're not very consistent about what stays and what is dropped during the build itself.